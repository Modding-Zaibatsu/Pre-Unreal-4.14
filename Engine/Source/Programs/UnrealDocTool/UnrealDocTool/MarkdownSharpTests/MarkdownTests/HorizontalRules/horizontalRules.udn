Paragraph.

--------------

Paragraph.

* * *

Paragraph.

***

Paragraph.

*****

Paragraph.

- - -

Paragraph.