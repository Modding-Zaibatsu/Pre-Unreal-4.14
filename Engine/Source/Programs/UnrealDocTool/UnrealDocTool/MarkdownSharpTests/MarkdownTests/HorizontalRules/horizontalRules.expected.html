<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" dir="ltr" lang="en-US">
    <head>
    </head>
    <body>
        <p>
            Paragraph.
        </p>
        <hr />
        <p>
            Paragraph.
        </p>
        <hr />
        <p>
            Paragraph.
        </p>
        <hr />
        <p>
            Paragraph.
        </p>
        <hr />
        <p>
            Paragraph.
        </p>
        <hr />
        <p>
            Paragraph.
        </p>
    </body>
</html>
